lifecycle-methods-lab/src/Form/Form.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+const Form = ({onSubmit}) => (
+
```

Comment:
to make it easier, the form could have had its own state of the values you want to send back on the parent component.